### PR TITLE
Adding post for cellAlias configuration

### PR DESCRIPTION
**Pull request by pratikgupta123**
Added blog post for cellAlias configuration and VTGate cross-cell behaviour

### CODE REVIEW

**Comment by pratikgupta123** [comment]
@deepthi 
@morgo 

This is the replacement PR for the previous one which was failing the DCO check.
Request you to please review this.

File: content/en/blog/2020-05-22-vitess-cross-cell.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  -mysql_server_port=$mysql_server_port \
+  -mysql_auth_server_impl=none
+```
+
+### Understanding VTGate Cross-Cell Behaviour
+
+1. VTGate will always be able to route to a master in a different cell (which is part of cells_to_watch) even without a cellAlias. the cellAlias requirement is only for replicas.
+2. By default VTGate sends all queries(reads and writes) to the master.
+3. Replica reads can be performed by first issuing a `use ks@replica` where `ks` should be replaced by your keyspace name.
+4. VTGate will always prefer tables that are in the same cell over tablets in other cells despite the CellAlias.
```
Comment (deepthi) [inline content/en/blog/2020-05-22-vitess-cross-cell.md]: One last typo: tables -> tablets

**Review by deepthi — APPROVED** [review]
LGTM. We can merge once the typo is fixed.

**Comment by morgo** [comment]
LGTM too. When you fix the typo, may I suggest updating the date to Tuesday 26th May? We can publish then.

File: content/en/blog/2020-05-22-vitess-cross-cell.md (hunk 1 above)
Comment (pratikgupta123) [reply content/en/blog/2020-05-22-vitess-cross-cell.md]: Thank you for spotting this, I have fixed it now.

**Comment by pratikgupta123** [comment]
Thank you @deepthi and @morgo for reviewing this.
I have fixed the typo and updated the date too as suggested.

**Comment by morgo** [comment]
(This is merged now, but will go live tomorrow.)

**Pull request merged**